dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyServer.java

```diff
@@ -94,10 +96,13 @@ protected void doOpen() throws Throwable {
         final NettyServerHandler nettyServerHandler = new NettyServerHandler(getUrl(), this);
         channels = nettyServerHandler.getChannels();
 
+        boolean keepalive = getUrl().getParameter(KEEP_ALIVE_KEY, Boolean.FALSE);
```

Comment:
shall default disable keep alive proper ?

Reply:
The default value of the keep alive parameter of netty is false, which can keep the configuration consistent with the previous version